features/logging_in_to_the_application.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Login to the app
+
+as a user
+in order to use the application
+I would like to login to my account
+
```

Comment:
Remove white spaces and format your code, please. Use gherkin formatter in your editor.